### PR TITLE
Game streak

### PR DESCRIPTION
**Pull request by thekevinWHYuan**
review please please please

### CODE REVIEW

File: backend/src/models/streakCalculation.ts [hunk 1]
```diff
@@ -3,19 +3,18 @@ import { User } from "./user.js";
 export async function calculateStreak(userID: string): Promise<number> {
   const user = await User.findById(userID);
 
-  const lastAnsweredTimestamp = user?.lastAnsweredTimestamp;
```
Comment (thekevinWHYuan) [inline backend/src/models/streakCalculation.ts:6]: I spent the at least 30 mins trying to figured out where was this used

**Comment by StarDylan** [comment]
How do we know if a user has answered a question today?

**Comment by thekevinWHYuan** [comment]
When I was looking through the code and the DB, the default lastAnswered timestamp everyone was given was set to null. There was a check I removed to see if lastAnswered timestamp was null or not and it would always return true regardless because I couldn't find the place in the frontend that made it not null.

**Comment by thekevinWHYuan** [comment]
Also, calculations are based off of the streakStart time anyways so I figured there was not point in having lastAnswered.

**Review by Anamooz — APPROVED** [review]

**Review by nrojo1124 — APPROVED** [review]

**Review by StarDylan — APPROVED** [review]

**Review by SanTran113 — APPROVED** [review]
LGTM!

**Pull request merged**